### PR TITLE
Resolve #5 "Implement TCP/IP socket function to send and recieve audio data"

### LINKED ISSUE #5: Implement TCP/IP socket function to send and recieve audio data
In this issue should be first found out if it is better to use one TCP/IP socket or two. **Microphone function can be changed in any way is needed.** 

If it is better to use one socket, audio data from microphone function should be sent from server to the client and from client to the server. 

Both server and client should be able to: 
- get audio data from microphone and send it;
- recieve audio data and play it back.

If it is better to use two sockets, then we actually need only two functions: one for client part and one for server part. Then we can simply run server and client parts of two sockets on one machine. Client should get audio data from the microphone and send it to server. Server should recieve this data and play it back.

### PR DESCRIPTION
**Pull request by anton2920**

### CODE REVIEW

File: logger.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import logging
+import os
```
Comment (Talkasi) [inline logger.py]: This include seems to be unused.

File: tcp.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    assert not log
+
+    log = logging.getLogger('TCP')
+
+
+def get_host_port(addr: str) -> (str, int):
+    addr_parts = addr.split(":")
+    if len(addr_parts) == 1:
+        # NOTE: adding default port value.
+        addr_parts.append(1234)
```
Comment (Talkasi) [inline tcp.py]: You are appending integer to a string. I think it is better to convert this value to string before appending.

File: logger.py (hunk 1 above)
Comment (anton2920) [reply logger.py]: Removed.

File: tcp.py (hunk 2 above)
Comment (anton2920) [reply tcp.py]: Changed.

**Pull request merged**